webrecorder/static/collection_info.js

```diff
@@ -20,7 +20,44 @@ var RecordingSelector = (function() {
         }
 
         BookmarksTable.filterByRecordings(getSelectedRecordingTitles());
-        updateRecordingFilterList(getSelectedRecordingIds());
+
+        var recordingIds = getSelectedRecordingIds();
+
+        updateRecordingFilterList(recordingIds);
+
+        if (event.originalEvent) {
+            updateUrl(recordingIds, $(this).attr("data-recording-id"));
+        }
+    }
+
+    $(window).on('popstate', function(event) {
+        if (!event.originalEvent.state) {
+            return;
+        }
+
+        var ids = event.originalEvent.state.ids;
+
+        $('.card').removeClass("card-selected");
```

Comment:
Also shouldn't need to directly call removeClass('.selected'), there is a toggleRecordingSelection method that encapsulates all that logic.  Or am I missing something?